src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportUnitTest.java

```diff
@@ -0,0 +1,43 @@
+package org.broadinstitute.hellbender.tools.genomicsdb;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.mockito.internal.util.io.IOUtil;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GenomicsDBImportUnitTest extends BaseTest{
+
+    @DataProvider
+    public Object[][] getBadTestFiles(){
+        return new Object[][]{
+                {"Sample1\tsamplePath\n"
+                +"Sample1\tsamplePath"},  //duplicate sample name
+                {""}, // empty file
+                {"Sample1\tSample2\tFile"}, //various wrong numbers of inputs
+                {"Sample1\t"},
+                {"Sample1"}
```

Comment:
Also test:

* Empty first token, non-empty second token
* First token only whitespace
* Extra tab at end of otherwise normal line
* Sample and samplePath separated by a newline instead of a tab
* A line with only tabs